src/components/Fields/PasswordField/PasswordField.tsx

```diff
@@ -23,7 +23,9 @@ const PasswordField: FC<PasswordFieldProps> = ({
   defaultValue,
   dataAutomation,
   strength,
+  strengthMessage = "Password must be at least 8 characters long, including uppercase letters and special characters",
```

Comment:
I think we have changed the validation rules, as far as I know, Sasha must have re-written them

Reply:
fixed